--- a/etc/config
+++ b/etc/config
@@ -2,8 +2,11 @@
 #
 # Please see http://i3wm.org/docs/userguide.html for a complete reference!
 
+# This defines which key super maps to on your keyboard.  
+# Alt key is Mod1, and Windows key is Mod4
 set $mod Mod4
 
+# This is the Solarized color palette.
 set $base03    #002b36
 set $base02    #073642
 set $base01    #586e75
@@ -23,16 +26,29 @@
 set $transparent #00000000
 set $lucent    #00000088
 
-set $ws1  "1   "
-set $ws2  "2   "
-set $ws3  "3   "
-set $ws4  "4   "
-set $ws5  "5   "
-set $ws6  "6   "
-set $ws7  "7   "
-set $ws8  "8   "
-set $ws9  "9   "
-set $ws10 "0   "
+# This is the labels which describe each i3 workspace.
+set $ws1   " 1 <span foreground='#268bd2'></span> "
+set $ws2   " 2 <span foreground='#2aa198'></span> "
+set $ws3   " 3 <span foreground='#859900'></span> "
+set $ws4   " 4 <span foreground='#b58900'></span> "
+set $ws5   " 5 <span foreground='#cb4b10'></span> "
+set $ws6   " 6 <span foreground='#dc322f'></span> "
+set $ws7   " 7 <span foreground='#d33682'></span> "
+set $ws8   " 8 <span foreground='#6c71c4'></span> "
+set $ws9   " 9 <span foreground='#93a1a1'></span>  "
+set $ws10  "10 <span foreground='#268bd2'></span>  "
+set $ws11  "11 <span foreground='#2aa198'></span>  "
+set $ws12  "12 <span foreground='#859900'></span>  "
+set $ws13  "13 <span foreground='#b58900'></span>  "
+set $ws14  "14 <span foreground='#cb4b10'></span>  "
+set $ws15  "15 <span foreground='#dc322f'></span>  "
+set $ws16  "16 <span foreground='#d33682'></span>  "
+set $ws17  "17 <span foreground='#6c71c4'></span>  "
+set $ws18  "18 <span foreground='#93a1a1'></span>  "
+set $ws19  "19 <span foreground='#268bd2'></span>  "
+
+# Invisible workspace
+set $ws20 " "
 
 # Use Mouse+$mod to drag floating windows to their wanted position
 floating_modifier $mod
@@ -40,7 +56,10 @@
 hide_edge_borders smart
 
 # start a terminal
-bindsym $mod+Return exec st
+bindsym $mod+Return exec /usr/bin/st
+
+# start a web browser
+bindsym $mod+Shift+Return exec /usr/bin/sensible-browser
 
 # kill focused window
 bindsym $mod+Shift+q kill
@@ -48,24 +67,32 @@
 # rofi app and window launchers
 bindsym $mod+space exec rofi -show drun -theme regolith-theme
 bindsym $mod+Shift+space exec rofi -show run -theme regolith-theme
-bindsym $mod+w exec rofi -show window -theme regolith-theme
+bindsym $mod+Ctrl+space exec rofi -show window -theme regolith-theme
 
-# change focus (try with vim keybiding for practice vim editor) (i3wm keybinding movement scheme conflict with vim wich will be disaster to use i3wm+vim together)
+# change focus
 bindsym $mod+Left focus left
 bindsym $mod+Down focus down
 bindsym $mod+Up focus up
 bindsym $mod+Right focus right
 
-# alternatively, you can use the cursor keys:
+bindsym $mod+h focus left
+bindsym $mod+j focus down
+bindsym $mod+k focus up
+bindsym $mod+l focus right
+
+# move windows in workspaces
 bindsym $mod+Shift+Left move left
 bindsym $mod+Shift+Down move down
 bindsym $mod+Shift+Up move up
 bindsym $mod+Shift+Right move right
 
-# split in horizontal orientation
-bindsym $mod+h split h
-# split in vertical orientation
-bindsym $mod+v split v
+bindsym $mod+Shift+h move left
+bindsym $mod+Shift+j move down
+bindsym $mod+Shift+k move up
+bindsym $mod+Shift+l move right
+
+# toggle split orientation
+bindsym $mod+BackSpace split toggle
 
 # enter fullscreen mode for the focused container
 bindsym $mod+f fullscreen toggle
@@ -74,19 +101,14 @@
 bindsym $mod+minus gaps inner current minus 6
 bindsym $mod+plus gaps inner current plus 6
 
-# change container layout (stacked, tabbed, toggle split)
-bindsym $mod+s layout stacking
-bindsym $mod+t layout tabbed
-bindsym $mod+l layout toggle split
-
 # show/hide keybindings
-bindsym $mod+question exec /usr/bin/conky-toggle.sh
+bindsym $mod+Shift+question exec /usr/bin/conky-toggle.sh
 
 # toggle tiling / floating
 bindsym $mod+Shift+f floating toggle
 
 # change focus between tiling / floating windows
-bindsym $mod+Alt+space focus mode_toggle
+bindsym $mod+Shift+t focus mode_toggle
 
 # switch to workspace
 bindsym $mod+1 workspace $ws1
@@ -99,10 +121,21 @@
 bindsym $mod+8 workspace $ws8
 bindsym $mod+9 workspace $ws9
 bindsym $mod+0 workspace $ws10
+bindsym $mod+Ctrl+1 workspace $ws11
+bindsym $mod+Ctrl+2 workspace $ws12
+bindsym $mod+Ctrl+3 workspace $ws13
+bindsym $mod+Ctrl+4 workspace $ws14
+bindsym $mod+Ctrl+5 workspace $ws15
+bindsym $mod+Ctrl+6 workspace $ws16
+bindsym $mod+Ctrl+7 workspace $ws17
+bindsym $mod+Ctrl+8 workspace $ws18
+bindsym $mod+Ctrl+9 workspace $ws19
+
+bindsym $mod+grave workspace $ws20
 
 # cycle between workspace
-bindsym $mod+n workspace next
-bindsym $mod+p workspace prev
+bindsym $mod+Tab workspace next
+bindsym $mod+Shift+Tab workspace prev
 
 # move focused container to workspace
 bindsym $mod+Shift+1 move container to workspace $ws1
@@ -115,6 +148,17 @@
 bindsym $mod+Shift+8 move container to workspace $ws8
 bindsym $mod+Shift+9 move container to workspace $ws9
 bindsym $mod+Shift+0 move container to workspace $ws10
+bindsym $mod+Shift+Ctrl+1 move container to workspace $ws11
+bindsym $mod+Shift+Ctrl+2 move container to workspace $ws12
+bindsym $mod+Shift+Ctrl+3 move container to workspace $ws13
+bindsym $mod+Shift+Ctrl+4 move container to workspace $ws14
+bindsym $mod+Shift+Ctrl+5 move container to workspace $ws15
+bindsym $mod+Shift+Ctrl+6 move container to workspace $ws16
+bindsym $mod+Shift+Ctrl+7 move container to workspace $ws17
+bindsym $mod+Shift+Ctrl+8 move container to workspace $ws18
+bindsym $mod+Shift+Ctrl+9 move container to workspace $ws19
+
+bindsym $mod+Shift+grave move container to workspace $ws20
 
 # reload the configuration file
 bindsym $mod+Shift+c reload
@@ -122,12 +166,24 @@
 # restart i3 inplace (preserves your layout/session, can be used to upgrade i3)
 bindsym $mod+Shift+r restart
 
+# exit i3
 bindsym $mod+Shift+e exit
 
+# lock the screen
 bindsym $mod+Escape exec gnome-screensaver-command --lock
 
+# put the system to sleep
 bindsym $mod+Shift+s exec systemctl suspend
 
+# shortcuts for common system configuration tasks
+bindsym $mod+c exec gnome-control-center
+bindsym $mod+w exec gnome-control-center wifi
+bindsym $mod+b exec gnome-control-center bluetooth
+bindsym $mod+d exec gnome-control-center display
+bindsym $mod+n exec gnome-control-center network
+bindsym $mod+p exec gnome-control-center power
+bindsym $mod+s exec gnome-control-center sound
+
 # resize window (you can also use the mouse for that)
 mode "resize" {
         # These bindings trigger as soon as you enter the resize mode
@@ -142,7 +198,6 @@
         bindsym Return mode "default"
         bindsym Escape mode "default"
 }
-
 bindsym $mod+r mode "resize"
 
 # Disable titlebar
@@ -167,25 +222,18 @@
 focus_follows_mouse no
 
 # Configure the bar
-
 bar {
-  mode hide
-  # font pango:Source Code Pro 20, FontAwesome 8
-  # font pango:DejaVu Sans Mono 10
   font pango:Source Code Pro Medium 13
   separator_symbol " "
   status_command i3blocks
   tray_output none
-  # i3bar_command i3bar -t
-  # height 36
-  # strip_workspace_numbers yes
 
   colors {
       background $base03
       statusline $base1
       separator  $blue
 #                        BORDER  BACKGROUND TEXT
-      focused_workspace  $base02  $base02  $blue
+      focused_workspace  $base02  $base02  $base2
       active_workspace   $base02  $base02  $base01
       inactive_workspace $base03  $base03  $base01
       urgent_workspace   $red     $red     $base3
@@ -194,11 +242,18 @@
 
 # Run other programs with i3
 
+# Composite manager
 exec --no-startup-id compton -f --config /etc/xdg/compton.conf
 
+# Lanuch the shortcut Conky window on first login
 exec --no-startup-id /usr/bin/regolith-ftue.sh
 
+# Hide the mouse pointer if unused for a duration
 exec --no-startup-id /usr/bin/unclutter -b
 
+# Draw the desktop background.  Randomly select from a series of photos starting with ESP_
 exec_always --no-startup-id feh --bg-scale --random /usr/share/backgrounds/ESP_*
 
+# Refresh wallpaper when monitor configuration changes.
+exec --no-startup-id /usr/bin/xeventbind resolution /usr/bin/wallpaper-refresh.sh
+
